tests/cloudpickle_test.py

```diff
@@ -2352,6 +2371,71 @@ def test_lookup_module_and_qualname_stdlib_typevar():
     assert name == 'AnyStr'
 
 
+def test_lookup_module_and_qualname_external_module_remove():
```

Comment:
(nitpick)
```suggestion
def test_register_pickle_by_value()
```
Also, maybe the next test could be merged into this one with an informative comment to explain the two tested behaviors.